app/pyproject.toml

```diff
@@ -3,11 +3,16 @@ DJANGO_SETTINGS_MODULE = "sampl.settings_dev"
 addopts = "-vv"
 
 [tool.pylint.messages_control]
-disable = "C0330, C0326"
+disable = "R, C, E, W1618, W0511"
 
 [tool.pylint.format]
 max-line-length = "88"
 
+[tool.pylint.master]
```

Comment:
why is pylint ignoring tests?

Reply:
I just had it skip the tests since there is a lot of ignores I need to add since it doesn't understand pytest fixtures... I think